components/konflux-ui/production/base/proxy/nginx.conf

```diff
@@ -141,6 +146,40 @@ http {
             include /mnt/nginx-generated-config/auth.conf;
         }
 
+        # GET requests to /api/k8s/api/v1/namespaces and /api/k8s/api/v1/namespaces/
+        # are handled from the namespace-lister.
+        # Requests with other methods are handled by the Kube-API
+        location = /api/k8s/api/v1/namespaces {
+            try_files /dev/null $ns_target;
+        }
+        location = /api/k8s/api/v1/namespaces/ {
+            try_files /dev/null $ns_target;
+        }
+
+        location @namespacelister {
+            auth_request_set $email  $upstream_http_x_auth_request_email;
+            auth_request /oauth2/auth;
+            proxy_read_timeout 30m;
+            proxy_set_header X-Email $email;
+
+            rewrite ^.*$ /api/v1/namespaces break;
+
+            proxy_pass https://namespace-lister.namespace-lister.svc.cluster.local:8080;
+        }
+
+        location @kubeapi {
+            auth_request_set $email  $upstream_http_x_auth_request_email;
+            auth_request /oauth2/auth;
+            proxy_read_timeout 30m;
+            proxy_set_header X-Email $email;
+
+            rewrite ^/api/k8s/(.*)/$ /$1 break;
+
+            proxy_pass https://kubernetes.default.svc;
+            proxy_set_header Impersonate-User $email;
+            include /mnt/nginx-generated-config/bearer.conf;
```

Comment:
The bearer.conf file is not present in production

https://github.com/redhat-appstudio/infra-deployments/blob/main/components%2Fkonflux-ui%2Fproduction%2Fbase%2Fproxy%2Fproxy.yaml#L66